lib/matplotlib/tests/test_text.py

```diff
@@ -886,3 +886,74 @@ def call(*args, **kwargs):
     # Every string gets a miss for the first layouting (extents), then a hit
     # when drawing, but "foo\nbar" gets two hits as it's drawn twice.
     assert info.hits > info.misses
+
+
+def test_set_antialiased():
+    txt = Text(.5, .5, "foo\nbar")
+    assert txt._antialiased == mpl.rcParams['text.antialiased']
+
+    txt.set_antialiased(True)
+    assert txt._antialiased is True
+
+    txt.set_antialiased(False)
+    assert txt._antialiased is False
+
+
+def test_get_antialiased():
+
+    txt2 = Text(.5, .5, "foo\nbar", antialiased=True)
+    assert txt2._antialiased is True
+    assert txt2.get_antialiased() == txt2._antialiased
+
+    txt3 = Text(.5, .5, "foo\nbar", antialiased=False)
+    assert txt3._antialiased is False
+    assert txt3.get_antialiased() == txt3._antialiased
+
+    txt4 = Text(.5, .5, "foo\nbar")
+    assert txt4.get_antialiased() == mpl.rcParams['text.antialiased']
+
+
+def test_annotation_antialiased():
+    annot = Annotation("foo\nbar", (.5, .5), antialiased=True)
+    assert annot._antialiased is True
+    assert annot.get_antialiased() == annot._antialiased
+
+    annot2 = Annotation("foo\nbar", (.5, .5), antialiased=False)
+    assert annot2._antialiased is False
+    assert annot2.get_antialiased() == annot2._antialiased
+
+    annot3 = Annotation("foo\nbar", (.5, .5), antialiased=False)
+    annot3.set_antialiased(True)
+    assert annot3.get_antialiased() is True
+    assert annot3._antialiased is True
+
+    annot4 = Annotation("foo\nbar", (.5, .5))
+    assert annot4._antialiased == mpl.rcParams['text.antialiased']
+
+
+@check_figures_equal()
+def test_text_antialiased_off_default_vs_manual(fig_test, fig_ref):
```

Comment:
I think this test and the one below should instead use figtext() so that it doesn't get bothered by tick labels, and just compares figtext+rcparams vs figtext+kwarg.

Reply:
Thanks for the suggestions! I rewrote the tests using `Figure.text()` and I think it's very similar to the `figtext()`as suggested.

Background:
Whether we are using antialiasing for tick labels will be determined at the time the figure is created, so we don't need to care about the tick labels. (This is true in most cases - if we are not overwriting `axes` with `Figure.subplots()` or something similar. To be more precise, determined at the time `axes` are created)